test/unit/test_TheorySimplifications.cc

```diff
@@ -169,46 +165,66 @@ TEST_F(ApplySubstitutionTest, test_BoolAtomSub) {
     PTRef a = logic.mkBoolVar("a");
     PTRef b = logic.mkBoolVar("b");
     PTRef fla = logic.mkAnd(a, logic.mkNot(b));
-    MapWithKeys<PTRef, PtAsgn, PTRefHash> subst;
-    subst.insert(b, PtAsgn{logic.getTerm_true(), l_True});
+    Logic::SubstMap subst;
+    subst.insert(b, logic.getTerm_true());
     PTRef res = Substitutor(logic, subst).rewrite(fla);
     EXPECT_EQ(res, logic.getTerm_false());
+    Logic::SubstMap subst2;
+    subst2.insert(b, logic.getTerm_true());
+    res = Substitutor(logic, subst2).rewrite(fla);
+    EXPECT_EQ(res, logic.getTerm_false());
```

Comment:
Aren't lines 172-175 doing the same thing as lines `168-171`? What is the purpose of the second block?